### PR TITLE
[15.0][MIG] partner_address_street3

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
Replace #1173

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
I don't know why there is an issue with the pre-commit
Any help?

**Comment by rousseldenis** [comment]
> I don't know why there is an issue with the pre-commit Any help?

Have you installed pre-commit before committing locally ?

**Comment by rousseldenis** [comment]
@emagdalenaC2i For the next time, if there is already published code (the other migration PR), it's better to keep authorship (cherry-pick) and then fix if needed.

**Comment by emagdalenaC2i** [comment]
@rousseldenis 
Of course, I have install and run the pre-commit as it is set in the full workflow here https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0#technical-method-to-migrate-a-module-from-140-to-150-branch

I am not sure if there is anything to cherry-pick in the previous PR. I mean, there is 2 commits in #1173:

1. black, isort, prettier: I have seen that the version number is changed in this commit, but I figure out that the rest are just prettier and format changes. 
3. And this one https://github.com/OCA/partner-contact/pull/1173/commits/3905c4baafaa7c59ce89c4de28c2fcd563f4dd55, that only add a line in the contributors file

Edit: Ok, I just saw the rest of the first commit and there is nothing to add here.

**Comment by jcdrubay** [comment]
@rousseldenis Do you know what needs to be done for this PR to be processed?

**Review by jcdrubay — APPROVED** [review]
Tested on Runboat, works as expected

**Comment by JuanyDForgeflow** [comment]
@jcdrubay Can you run pre-commit? seems like that's what's missing

**Comment by jcdrubay** [comment]
Let's see how this goes: https://github.com/OCA/partner-contact/pull/1292

**Comment by rousseldenis** [comment]
@emagdalenaC2i Do you mind if we switch there ?

File: setup/partner_address_street3/setup.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import setuptools
+
+setuptools.setup(
+    setup_requires=['setuptools-odoo'],
+    odoo_addon=True,
+)
```
Comment (xhnilic3) [inline setup/partner_address_street3/setup.py:6]: Maybe I would add a new line at the end of the file

**Pull request closed without merging**